packages/ozone/src/lexicon/lexicons.ts

```diff
@@ -11179,6 +11179,69 @@ export const schemaDict = {
               type: 'string',
             },
           },
+          accountStats: {
```

Comment:
Instead of having them spread all across the `subjectStatusView` object, we should consider a container object for `repoStats` or `repoModerationStats` ?